### DISCUSSION TITLE
Several Editor Improvements

### DISCUSSION THREAD

**Discussion by Paladynee** [ppy/osu#30723] [Ideas]
## Shortcut for Keybinds  
It would be beneficial to include a `Help > Shortcuts` button in the editor page that redirects to `Options > Input > Configure > Global > Editor`. This would serve as both a quick reference guide for keybinds and an easy way to configure them.

## Keybind Tooltips on Buttons  
Displaying tooltips with the current keybinds for editor functions would improve usability. For instance, if the `W` key is set to activate `Whistle`, hovering over the `Whistle` button should show a tooltip like `Current Keybind: W`.  

**Comparison of behaviors:**  
- **Stable version:**  
  ![image](https://github.com/user-attachments/assets/11f95a2d-49de-4326-8976-f8b089318254)  

- **Lazer version:**  
  ![image](https://github.com/user-attachments/assets/261bc67f-f9c5-40e0-9ee8-c94b18c437a9)  

## Right-Click for New Combo  
In the stable version, right-clicking an empty space toggles the `New Combo` state. This functionality is missing in Lazer, requiring users to unlearn a familiar shortcut. While the omission may be intentional to accommodate new right-click functions in Lazer, reintroducing this feature as an optional setting could be helpful. Since toggling `New Combo` is non-destructive, accidental activation shouldn't pose significant issues.

## Bottom Bar Enhancements  
The bottom timeline markers in the stable version were a "1-pixel wide" design, which allowed for precise navigation. The newer rounded markers in Lazer introduce some ambiguity in navigation. Additionally, the "green lines" in the stable version were a useful visual aid for navigation and are absent in Lazer.  

Another critical issue is the lack of bookmark support in the Lazer editor, further reducing functionality in the bottom bar.  

**Feature comparison:**  
- **Stable version:**  
  ![image](https://github.com/user-attachments/assets/4f276b90-0e9e-484d-89a1-e74f50414e7b)  

- **Lazer version:**  
  ![image](https://github.com/user-attachments/assets/5269a92b-6192-47f8-af49-df5974af2340)  

The bottom bar in Lazer has reduced navigability and functionality compared to the stable version.

**Comment by rvnexe** [comment]
No need to hover! Just like how we get hotkey hint in other games. Something like this:
![image](https://github.com/user-attachments/assets/d2bddf61-050e-44be-8110-bbcbd5a5b1e0)

**Comment by Paladynee** [comment]
@peppy mentioning since it's been a month since i opened this discussion.

**Reply by peppy** [reply]
There's hundreds of editor suggestions. I don't disagree with yours, but it's a matter of them getting the time of day to see implemented.
